Readme.org

```diff
@@ -707,14 +720,33 @@
    custom-id, as the examples below:
 
    * [[Tips]].
-   * [[#my-heading][Heading]] with a =CUSTOM_ID= property.
+   * [[#my-heading][Heading]] with a =CUSTOM_ID= property.q
```

Comment:
typo?

Reply:
Fixed in https://github.com/yjwen/org-reveal/commit/24cb9276d37e1a123ae52eb58f41db150a1861d2